consensus/src/round_manager.rs

```diff
@@ -525,7 +582,9 @@ impl RoundManager {
         if message_round < self.round_state.current_round() {
             return Ok(false);
         }
-        self.sync_up(sync_info, author).await?;
+        let sync_info_verified = self.sync_up_commit_certs(sync_info, author).await?;
```

Comment:
the var name is a bit confusing, I thought it's a sync info that's already verified, probably should call it `is_sync_info_verified`